src/pages/Home.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import TopContainer from "../components/TopContainer/TopContainer";
+import FilterPanel from "../components/FilterPanel/FilterPanel";
+import "../styles/pages/home.scss";
+
+const Home = () => {
+    return(
+        <>
+            <TopContainer />
+            <FilterPanel />
+            <div className="movies-container">
+                <h2>No movie found</h2>
+            </div>
```

Comment:
I think it should be a part of `MovieList` component. If there is no movies -> `"No movies found"` should be shown

Reply:
fixed